libcxx/test/libcxx/ranges/range.adaptors/range.stride.view/iterator/dereference.pass.cpp

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+// UNSUPPORTED: no-exceptions
+// UNSUPPORTED: libcpp-hardening-mode=none
+// XFAIL: availability-verbose_abort-missing
+
+#include "check_assertion.h"
+#include <ranges>
+
+int main(int, char**) {
+  {
+    int range[]   = {1, 2, 3};
```

Comment:
Please test for a variety of ranges, including non-contiguous containers (this is applicable to all tests).

Reply:
Started to work on this in 10809679a823d8286b7bd45d3dbf9c23b2e4c4d5.